leftwm-core/src/event_loop.rs

```diff
@@ -27,12 +27,18 @@ impl<C: Config, SERVER: DisplayServer> Manager<C, SERVER> {
     /// # Errors
     /// `EventResponse` if the initialisation of the command pipe or/and the state socket failed.
     pub async fn event_loop(mut self) -> Result<(), Error> {
-        let mut state_socket = get_state_socket().await?;
-        let mut command_pipe = get_command_pipe().await?;
+        let state_socket = get_state_socket().await?;
+        let command_pipe = get_command_pipe().await?;
 
         self.call_up_scripts();
+        self.start_event_loop(state_socket, command_pipe).await
+    }
 
-        // Main event loop.
+    async fn start_event_loop(
```

Comment:
isn't the naming here just backwards with this commit?

Reply:
huh, what do you mean with this?

Reply:
```rust
pub async fn start_event_loop(mut self) -> Result<(), Error> {
        let state_socket = get_state_socket().await?;
        let command_pipe = get_command_pipe().await?;

        self.call_up_scripts();
        self.event_loop(state_socket, command_pipe).await
    }

    async fn event_loop(
```

Reply:
ooh, yeah, I can do that quickly

Reply:
done!